src/Components/Button.js

```diff
@@ -34,35 +33,28 @@ const Button = ({
   buttonColor,
   textColor,
 }: $Component & Props) => (
-  <div
-    className={classes.root}
-    style={alignment ? { textAlign: alignment } : null}
+  <EUIButton
+    element="a"
+    href={href}
+    color={color || "primary"}
+    onClick={(e) => e.preventDefault()}
+    style={{
+      paddingTop: buttonPaddingTop ? `${buttonPaddingTop}px` : "0",
+      paddingRight: buttonPaddingRight ? `${buttonPaddingRight}px` : "0",
+      paddingBottom: buttonPaddingBottom ? `${buttonPaddingBottom}px` : "0",
+      paddingLeft: buttonPaddingLeft ? `${buttonPaddingLeft}px` : "0",
+      margin: "0 auto",
```

Comment:
What's up with this margin?

Reply:
The button wouldn't align l/c/r without it, the other option seemed to wrap it in a div, but that might be against what we are trying to do here? Any thoughts?